db/migrate/20160603182320_add_en_passant_file_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnPassantFileToGames < ActiveRecord::Migration
+  def change
+    add_column :games, :en_passant_file, :integer
```

Comment:
I'm a bit confused by the naming here -- does `file` refer to a position on the board, as in "rank and file"? (My knowledge of chess might be sketchier than I thought 😄)